package.json

```diff
@@ -25,16 +25,13 @@
   "dependencies": {
     "@actions/core": "^1.2.0",
     "@actions/exec": "^1.0.1",
-    "@actions/glob": "^0.1.0",
-    "@actions/http-client": "^1.0.8",
     "@actions/io": "^1.0.1",
-    "uuid": "^3.3.3"
+    "@actions/cache": "^0.1.0"
```

Comment:
Let's make sure we publish a `v1` of `@actions/cache` before we publish this `v2` action. Would be odd if we had a major action version consume a preview package